controllers/controllers.js

```diff
@@ -21,9 +21,19 @@ exports.getArticleById = (req, res, next) => {
             if (article !== undefined) {
                 res.status(200).send({ article })
             }
-            else res.status(404).send({
-                Message: "Article ID not found."
-            })
         })
-        .catch(next)
+        .catch((error) => {
+            next(error)
+        })
 }
+exports.getCommentsByArticle = (req, res, next) => {
+    selectCommentsByArticle(req)
+        .then((comments) => {
+            if (comments !== undefined) {

```

Comment:
Hopefully we shouldn't need this condition here, if we are passing data back to the controller that should be because it IS defined. Any error handling should be dealt with in the model (if the response happens to be undefined for any reason)